### ISSUE TITLE
buddhist neural network

### ISSUE THREAD

**Issue by silky** [silky/ideas#467]
it features significant self-attention and self-awareness

it performs classification, but predicts the same class for every input as it has a nondual mind

it's loss function features a single term for each of the 6 [paramitas](https://en.wikipedia.org/wiki/P%C4%81ramit%C4%81):

- Generosity: to cultivate the attitude of generosity.
- Discipline: refraining from harm.
- Patience: the ability not to be perturbed by anything.
- Diligence: to find joy in what is virtuous, positive or wholesome.
- Meditative concentration: not to be distracted.
- Wisdom: the perfect discrimination of phenomena, all knowable things.
